reference/7.1/Microsoft.PowerShell.Utility/Add-Type.md

```diff
@@ -445,6 +445,11 @@ The acceptable values for this parameter are as follows:
 - Library
 - WindowsApplication
 
+> [!IMPORTANT]
+> As of PowerShell 7.1, `ConsoleApplication` and `WindowsApplication` are not supported and
+> PowerShell will throw a terminating error if they are specified as values for the **OutputType**
```

Comment:
Can we remove the passive and change "will throw" to "throws"? :)  Also, the "if they are specified" should be "if either are specified", since someone wouldn't use both together.

Reply:
great catch! Fixed